include/lo2s/config.hpp

```diff
@@ -92,6 +92,9 @@ struct Config
     // syscalls
     bool use_syscalls = false;
     std::vector<int64_t> syscall_filter;
+    // NEC SX-Aurora Tsubasa
+    bool use_nec;
+    std::chrono::nanoseconds nec_readout_interval;
```

Comment:
is there a reason to call it readOUT_interval instead of read_interval like the others?